src/main/java/frc/robot/commands/commandgroups/ShootCG.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.commands.commandgroups;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.RobotContainer;
+import frc.robot.commands.GenericTurnToTargetCMD;
+import frc.robot.commands.MoveMovableSubsystem;
+import frc.robot.commands.PIDCommand;
+import frc.robot.constants.RobotConstants.*;
+import frc.robot.subsystems.shooter.ShooterCalculations;
+
+public class ShootCG extends ParallelCommandGroup {
+    public ShootCG(RobotContainer robotContainer) {
+        PIDCommand shooterCMD = new PIDCommand(
+                robotContainer.shooterSS,
+                () -> ShooterCalculations.calculateVelocity(robotContainer.limelight.getTy()));
```

Comment:
make sure this works

Reply:
will be tested when we have a robot but in the mean time ill add a TODO